full/Dockerfile

```
@@ -263,7 +263,7 @@ RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
         rls \
         rust-analysis \
         rust-src \
-    && .cargo/bin/rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
+    && .cargo/bin/rustup completions bash cargo | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
```

Oops, my bad, it seems that `rustup completions bash cargo` actually only outputs completions for `cargo`, and that you still need to use `rustup completions bash` separately.

@JesterOrNot Could you please double-check?

In case both are needed, we could do it like this:

```suggestion
    && .cargo/bin/rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
    && .cargo/bin/rustup completions bash cargo | sudo tee /etc/bash_completion.d/rustup.cargo-bash-completion > /dev/null \
```